common/json/src/test/java/com/bazaarvoice/emodb/common/json/JsonStreamingArrayParserTest.java

```diff
@@ -67,19 +67,19 @@ public void testEOFException() {
     @Test
     @SuppressWarnings("unchecked")
     public void testMalformedChunkException() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS00112)